src/main/java/net/mcreator/ui/modgui/PlantGUI.java

```diff
@@ -517,6 +517,11 @@ else if (page == 1)
 			dyn.setIcon(TiledImageCache.plantGrowingYes);
 		else
 			dyn.setIcon(TiledImageCache.plantGrowingNo);
+
+		useLootTableForDrops.addActionListener(e -> {

```

Comment:
Just add the two lines from inside listener, in this function you just need to set enabled/disabled state, not register listener again